src/components/ApplicationGroup/AppGroup.types.ts

```diff
@@ -331,6 +337,11 @@ export interface AppGroupListType {
     clusterName: string
     environmentId: number
     apps: ApplistEnvType[]
+    description?: string
+    environmentType?: string
+    createdOn?: string
+    createdBy?: string
+    clusterId?: number
```

Comment:
- Pls check if all fields optional from the API
- Also check if we can better type the environmentType

Reply:
being used in multiple components, thats why have to make optional